project/src/components/app/app.tsx

```diff
@@ -11,10 +11,11 @@ import PrivateRoute from '../private-route/private-route';
 import { useAppSelector } from '../../hooks';
 import LoadingScreen from '../loading-screen/loading-screen';
 import Player from '../player-screen/player-screen';
+import { isCheckedAuth } from '../../film';
 
 function App(): JSX.Element {
-  const { isDataLoaded} = useAppSelector((state) => state);
-  if (!isDataLoaded) {
+  const { isDataLoaded, authorizationStatus} = useAppSelector((state) => state);
```

Comment:
какой же это селектор?
```suggestion
  const isDataLoaded = useAppSelector((state) => state.isDataLoaded);
  const authorizationStatus = useAppSelector((state) => state.authorizationStatus);
```

Reply:
селектор по возможность и должен возвращать минимально необходимое значение

Reply:
или так. но не глубже. react-redux сравнивает результаты из селекторов shallow equal и когда не равно - перерисовывает компонент
```suggestion
  const { isDataLoaded, authorizationStatus} = useAppSelector((state) => {
    const { isDataLoaded, authorizationStatus} = state;
    return { isDataLoaded, authorizationStatus };
  });
```